[feature] introduce postfix_extra_packages

trombik opened the pull request:
fixes #26

Commit:
remove Makefile, run postmap on changed tables

Commit:
make is not required anymore

Commit:
remove unused var

Commit:
also remove make from empty_aliases_spec.rb

Commit:
introduce postfix_extra_package

fixes #26

Commit:
QA

Commit:
pflogsumm is included in postfix-perl-scripts

Commit:
Merge branch 'master' into postfix_extra_package

Commit:
fix the spec

mitsururike reviewed (commented):
s/postfix_extra_package/postfix_extra_packages/ as it is an array.

Commit:
's/postfix_extra_package/postfix_extra_packages/g'

https://github.com/reallyenglish/ansible-role-postfix/pull/29#pullrequestreview-41118939
